lib/model-builder.js

```diff
@@ -348,13 +387,14 @@ ModelBuilder.prototype.define = function defineClass(className, properties, sett
    * ```
    *
    * @param {String} className Name of the new model being defined.
-   * @options {Object} properties Properties to define for the model, added to properties of model being extended.
-   * @options {Object} settings Model settings, such as relations and acls.
-   *
+   * @options {Object} subclassProperties child model properties, added to base model
+   *   properties.
+   * @options {Object} subclassSettings child model settings such as relations and acls,
+   *   merged with base model settings.
    */
   ModelClass.extend = function(className, subclassProperties, subclassSettings) {
-    var properties = ModelClass.definition.properties;
-    var settings = ModelClass.definition.settings;
+    var baseclassProperties = ModelClass.definition.properties;
+    var baseclassSettings = ModelClass.definition.settings;
```

Comment:
I like the idea of prefixing to make the intent more clear. Can we use `baseClass` case please? Or perhaps just `base` or `parent`, e.g. `baseProperties` or `parentProperties`. Or maybe `inherited`?